packages/live-status-gateway/src/wsHandler.ts

```diff
@@ -0,0 +1,109 @@
+import { StudioId } from '@sofie-automation/corelib/dist/dataModel/Ids'
+import { Observer } from '@sofie-automation/server-core-integration'
+import { Logger } from 'winston'
+import { WebSocket } from 'ws'
+import { CoreHandler } from './coreHandler'
+
+export abstract class WsTopicBase {
```

Comment:
I assume that you mean `WebSocket` with `Ws`?
Would be nice if abbreviations weren't used as it forces the reading developer to guess what is meant (and guessing leads to wrong guesses which leads to errors and/or increased development time). 
Here `Ws` could easily be either `WebSocket` or `WebServer`.

Also, what does `Topic` refere to? You have an interface called `WsTopic` but that interface is not used at all in the implementation of `WsTopicBase`, so this is essentially (and should be named) a `BaseWebSocket`

Reply:
Renamed in https://github.com/nrkno/sofie-core/pull/777/commits/fb6235a7b90c2f09f534a1a843aa1d222f6c5466

`WebSocketTopicBase` is extended by topics and provides common functionality via inheritance. `Topic` is an interface that allows us to store/access topics without knowing the class structure.